Examples/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Examples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var algo = new MeanReversionPaperOnly();
+            algo.Run();
+            Console.Read();
```

Comment:
I understand that it's just an example but this example demonstrates what we shouldn't close connections to NATS/REST API endpoint. AFAIU only one possible way for break Run method execution w/o waiting till end of trading session is to press Ctrl+C and stop thread w/o any cleanup attempt. Maybe it's OK for example but in this case you have to state in you Wiki/Docs that in the real code it's better to call Dispose on client objects for proper disconnect/resource cleanup.

Reply:
Well, initially I tried to handle the disposal with using() as you've done in the tests, but that actually doesn't work because it kills the subscriptions. (Some of those tests never would have passed during market hours because of that - no data ever came in.) So I just avoided messing with that since I didn't have another example to work off. Ctrl+C without cleanup is the intended way to stop scripts in our other language SDKs, so I don't really see an issue with that anyhow. Generally scripts are run on their own rather than as part of a larger program, and you can correct me if I'm wrong but I don't think it's an issue unless it's the latter case.